src/Symfony/Component/Mime/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Exception;
+
+interface ExceptionInterface extends \Throwable
```

Comment:
You should add `@author` here too, like in the following exception

Reply:
done